SwiftEntity.js

```diff
@@ -9,9 +10,10 @@ class SwiftEntity {
         this.authenticator = authenticator;
     }
 
-    list(extra) {
+    list(extra, query = '') {
+        const querystring = ( query ) ? '?'+queryString.stringify(query) : query;
```

Comment:
Just in case 'query' is zero, null or undefined:

const querystring = ( query ) ? '?'+queryString.stringify(query) : '';